Modules/Microsoft365DSC/DSCResources/MSFT_IntuneAntivirusPolicyWindows10SettingCatalog/MSFT_IntuneAntivirusPolicyWindows10SettingCatalog.psm1

```diff
@@ -354,7 +356,7 @@ function Get-TargetResource
         $returnHashtable.Add('CertificateThumbprint', $CertificateThumbprint)
         $returnHashtable.Add('ManagedIdentity', $ManagedIdentity.IsPresent)
 
-        return $returnHashtable
+        return ,$returnHashtable
```

Comment:
Is the comma an oversight?

Reply:
Yes, it is not required here, I'll update the PR

Reply:
done